src/apis.js

```diff
@@ -0,0 +1,21 @@
+function getData(path) {
```

Comment:
Do you need this helper function here? It doesn't look like it's saving much code.